MONGO_DB=
MONGO_USER=
MONGO_PASSWORD=
MONGO_PORT=5432

# REST_API
APP_WORKING_DIR=/app
REST_API_APP_PORT=5000
REST_API_OUTER_APP_PORT=5000






2024-05-09 02:52:33 Traceback (most recent call last):
2024-05-09 02:52:33   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:33     sys.exit(main())
2024-05-09 02:52:33              ^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:33     return self.main(*args, **kwargs)
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:33     rv = self.invoke(ctx)
2024-05-09 02:52:33          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:33     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:33     return __callback(*args, **kwargs)
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:33     run(
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:33     server.run()
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:33     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:33     return runner.run(main)
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:33     return self._loop.run_until_complete(task)
2024-05-09 02:52:33            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:33   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:34     return future.result()
2024-05-09 02:52:34            ^^^^^^^^^^^^^^^
2024-05-09 02:52:34   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:34     config.load()
2024-05-09 02:52:34   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:34     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:34                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:34   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:34     raise exc from None
2024-05-09 02:52:34   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:34     module = importlib.import_module(module_str)
2024-05-09 02:52:34              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:34   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:34     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:34            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:34   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:34   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:34   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:34   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:34   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:34   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:34   File "/main.py", line 7, in <module>
2024-05-09 02:52:34     from .modules.routers import add_data, search
2024-05-09 02:52:34 ImportError: attempted relative import with no known parent package
2024-05-09 02:52:42 Traceback (most recent call last):
2024-05-09 02:52:42   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:42     sys.exit(main())
2024-05-09 02:52:42              ^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:42     return self.main(*args, **kwargs)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:42     rv = self.invoke(ctx)
2024-05-09 02:52:42          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:42     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:42     return __callback(*args, **kwargs)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:42     run(
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:42     server.run()
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:42     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:42     return runner.run(main)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:42     return self._loop.run_until_complete(task)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:42     return future.result()
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:42     config.load()
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:42     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:42                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:42     raise exc from None
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:42     module = importlib.import_module(module_str)
2024-05-09 02:52:42              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:42     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:42            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:42   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:42   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:42   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:42   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:42   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:42   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:42   File "/main.py", line 7, in <module>
2024-05-09 02:52:42     from .modules.routers import add_data, search
2024-05-09 02:52:42 ImportError: attempted relative import with no known parent package
2024-05-09 02:52:45 Traceback (most recent call last):
2024-05-09 02:52:45   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:45     sys.exit(main())
2024-05-09 02:52:45              ^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:45     return self.main(*args, **kwargs)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:45     rv = self.invoke(ctx)
2024-05-09 02:52:45          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:45     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:45     return __callback(*args, **kwargs)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:45     run(
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:45     server.run()
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:45     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:45     return runner.run(main)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:45     return self._loop.run_until_complete(task)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:45     return future.result()
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:45     config.load()
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:45     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:45                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:45     raise exc from None
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:45     module = importlib.import_module(module_str)
2024-05-09 02:52:45              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:45     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:45            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:45   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:45   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:45   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:45   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:45   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:45   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:45   File "/main.py", line 7, in <module>
2024-05-09 02:52:45     from .modules.routers import add_data, search
2024-05-09 02:52:45 ImportError: attempted relative import with no known parent package
2024-05-09 02:52:48 Traceback (most recent call last):
2024-05-09 02:52:48   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:48     sys.exit(main())
2024-05-09 02:52:48              ^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:48     return self.main(*args, **kwargs)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:48     rv = self.invoke(ctx)
2024-05-09 02:52:48          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:48     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:48     return __callback(*args, **kwargs)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:48     run(
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:48     server.run()
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:48     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:48     return runner.run(main)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:48     return self._loop.run_until_complete(task)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:48     return future.result()
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:48     config.load()
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:48     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:48                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:48     raise exc from None
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:48     module = importlib.import_module(module_str)
2024-05-09 02:52:48              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:48     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:48            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:48   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:48   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:48   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:48   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:48   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:48   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:48   File "/main.py", line 7, in <module>
2024-05-09 02:52:48     from .modules.routers import add_data, search
2024-05-09 02:52:48 ImportError: attempted relative import with no known parent package
2024-05-09 02:52:52 Traceback (most recent call last):
2024-05-09 02:52:52   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:52     sys.exit(main())
2024-05-09 02:52:52              ^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:52     return self.main(*args, **kwargs)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:52     rv = self.invoke(ctx)
2024-05-09 02:52:52          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:52     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:52     return __callback(*args, **kwargs)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:52     run(
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:52     server.run()
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:52     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:52     return runner.run(main)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:52     return self._loop.run_until_complete(task)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:52     return future.result()
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:52     config.load()
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:52     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:52                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:52     raise exc from None
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:52     module = importlib.import_module(module_str)
2024-05-09 02:52:52              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:52     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:52            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:52   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:52   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:52   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:52   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:52   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:52   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:52   File "/main.py", line 7, in <module>
2024-05-09 02:52:52     from .modules.routers import add_data, search
2024-05-09 02:52:52 ImportError: attempted relative import with no known parent package
2024-05-09 02:52:56 Traceback (most recent call last):
2024-05-09 02:52:56   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:52:56     sys.exit(main())
2024-05-09 02:52:56              ^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:52:56     return self.main(*args, **kwargs)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:52:56     rv = self.invoke(ctx)
2024-05-09 02:52:56          ^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:52:56     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:52:56     return __callback(*args, **kwargs)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:52:56     run(
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:52:56     server.run()
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:52:56     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:52:56     return runner.run(main)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:52:56     return self._loop.run_until_complete(task)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:52:56     return future.result()
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:52:56     config.load()
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:52:56     self.loaded_app = import_from_string(self.app)
2024-05-09 02:52:56                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:52:56     raise exc from None
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:52:56     module = importlib.import_module(module_str)
2024-05-09 02:52:56              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:52:56     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:52:56            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:52:56   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:52:56   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:52:56   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:52:56   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:52:56   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:52:56   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:52:56   File "/main.py", line 7, in <module>
2024-05-09 02:52:56     from .modules.routers import add_data, search
2024-05-09 02:52:56 ImportError: attempted relative import with no known parent package
2024-05-09 02:53:02 Traceback (most recent call last):
2024-05-09 02:53:02   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:53:02     sys.exit(main())
2024-05-09 02:53:02              ^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:53:02     return self.main(*args, **kwargs)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:53:02     rv = self.invoke(ctx)
2024-05-09 02:53:02          ^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:53:02     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:53:02     return __callback(*args, **kwargs)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:53:02     run(
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:53:02     server.run()
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:53:02     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:53:02     return runner.run(main)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:53:02     return self._loop.run_until_complete(task)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:53:02     return future.result()
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:53:02     config.load()
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:53:02     self.loaded_app = import_from_string(self.app)
2024-05-09 02:53:02                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:53:02     raise exc from None
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:53:02     module = importlib.import_module(module_str)
2024-05-09 02:53:02              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:53:02     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:53:02            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:02   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:53:02   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:53:02   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:53:02   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:53:02   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:53:02   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:53:02   File "/main.py", line 7, in <module>
2024-05-09 02:53:02     from .modules.routers import add_data, search
2024-05-09 02:53:02 ImportError: attempted relative import with no known parent package
2024-05-09 02:53:10 Traceback (most recent call last):
2024-05-09 02:53:10   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:53:10     sys.exit(main())
2024-05-09 02:53:10              ^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:53:10     return self.main(*args, **kwargs)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:53:10     rv = self.invoke(ctx)
2024-05-09 02:53:10          ^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:53:10     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:53:10     return __callback(*args, **kwargs)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:53:10     run(
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:53:10     server.run()
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:53:10     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:53:10     return runner.run(main)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:53:10     return self._loop.run_until_complete(task)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:53:10     return future.result()
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:53:10     config.load()
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:53:10     self.loaded_app = import_from_string(self.app)
2024-05-09 02:53:10                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:53:10     raise exc from None
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:53:10     module = importlib.import_module(module_str)
2024-05-09 02:53:10              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:53:10     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:53:10            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:10   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:53:10   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:53:10   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:53:10   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:53:10   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:53:10   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:53:10   File "/main.py", line 7, in <module>
2024-05-09 02:53:10     from .modules.routers import add_data, search
2024-05-09 02:53:10 ImportError: attempted relative import with no known parent package
2024-05-09 02:53:24 Traceback (most recent call last):
2024-05-09 02:53:24   File "/usr/local/bin/uvicorn", line 8, in <module>
2024-05-09 02:53:24     sys.exit(main())
2024-05-09 02:53:24              ^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
2024-05-09 02:53:24     return self.main(*args, **kwargs)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
2024-05-09 02:53:24     rv = self.invoke(ctx)
2024-05-09 02:53:24          ^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
2024-05-09 02:53:24     return ctx.invoke(self.callback, **ctx.params)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
2024-05-09 02:53:24     return __callback(*args, **kwargs)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 404, in main
2024-05-09 02:53:24     run(
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/main.py", line 569, in run
2024-05-09 02:53:24     server.run()
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 60, in run
2024-05-09 02:53:24     return asyncio.run(self.serve(sockets=sockets))
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
2024-05-09 02:53:24     return runner.run(main)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
2024-05-09 02:53:24     return self._loop.run_until_complete(task)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
2024-05-09 02:53:24     return future.result()
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 67, in serve
2024-05-09 02:53:24     config.load()
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 477, in load
2024-05-09 02:53:24     self.loaded_app = import_from_string(self.app)
2024-05-09 02:53:24                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 24, in import_from_string
2024-05-09 02:53:24     raise exc from None
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
2024-05-09 02:53:24     module = importlib.import_module(module_str)
2024-05-09 02:53:24              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-05-09 02:53:24     return _bootstrap._gcd_import(name[level:], package, level)
2024-05-09 02:53:24            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-05-09 02:53:24   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-05-09 02:53:24   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-05-09 02:53:24   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-05-09 02:53:24   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-05-09 02:53:24   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-05-09 02:53:24   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-05-09 02:53:24   File "/main.py", line 7, in <module>
2024-05-09 02:53:24     from .modules.routers import add_data, search
2024-05-09 02:53:24 ImportError: attempted relative import with no known parent package
